### ISSUE TITLE
Varnish 5.2.1 on Ubuntu 18.04: Assert error in h2_do_req(), http2/cache_http2_proto.c line 496

### ISSUE THREAD

**Issue by ffh-webadmin** [varnishcache/varnish-cache#2583]
We tested the current Ubuntu package + Hitch 1.4.4 on upcomming 18.04. After two minutes in production, syslog is flooded by messages like this

Feb 23 12:07:56 cache1-vm varnishd[3140]: "Missing errorhandling code in h2_rxframe(), http2/cache_http2_proto.c line 535:
Feb 23 12:07:56 cache1-vm varnishd[3140]: Child (4547) Panic at: Fri, 23 Feb 2018 11:07:56 GMT#012"Missing errorhandling code in h2_rxframe(), http2/cache_http2_proto.c line 535:#012  Condition(h2->rxf_stream > h2->highest_stream) not true.thread = (cache-worker)#012version = varnish-5.0.0 revision 99d036f#012ident = Linux,4.13.0-36-generic,x86_64,-junix,-smalloc,-smalloc,-hcritbit,epoll#012Backtrace:#012  0x559691aba6df: varnishd(+0x426df) [0x559691aba6df]#012  0x559691ae5091: varnishd(+0x6d091) [0x559691ae5091]#012  0x559691ad4413: varnishd(+0x5c413) [0x559691ad4413]#012  0x559691ad4a7b: varnishd(+0x5ca7b) [0x559691ad4a7b]#012  0x7efe77ef67fc: libpthread.so.0(+0x77fc) [0x7efe77ef67fc]#012  0x7efe77c23b5f: libc.so.6(clone+0x3f) [0x7efe77c23b5f]#012req = 0x7efe635ff020 {#012  vxid = 2228294, transport = H2#012  step = 0x0,#012  req_body = R_BODY_INIT,#012  err_code = 1, err_reason = (null),#012  restarts = 0, esi_level = 0,#012  sp = 0x7efe648c3420 {#012    fd = 215, vxid = 2228294,#012    t_open = 1519384074.928511,#012    t_idle = 1519384074.985065,#012    transport = H2 {#012      streams {#012        0x00000000 idle  #012      }#012    }#012client = 77.87.228.71 2467,#012  },#012  ws = 0x7efe635ff1f8 {#012    id = \"req\",#012    {s, f, r, e} = {0x7efe63600ff8, +296, (nil), +57344},#012  },#012  http_conn = 0x7efe635ff128 {#012    fd = 215,#012    doclose = NULL,#012    ws = 0x7efe67154e28,#012    {rxbuf_b, rxbuf_e} = {0x7efe671543f0, 0x7efe671543fd},#012    {pipeline_b, pipeline_e} = {(nil), (nil)},#012    content_length = 0,#012    body_status = none,#012    first_byte_timeout = 0.000000,#012    between_bytes_timeout = 0.000000,#012  },#012  http[req] = 0x7efe635ff290 {#012    ws[] = (nil),#012    hdrs {#012    },#012  },#012  flags = {#012  },#012},#012#012"

And

Feb 23 13:12:49 cache1-vm varnishd[1094]: Child (1855) Panic at: Fri, 23 Feb 2018 12:12:49 GMT#012Assert error in h2_do_req(), http2/cache_http2_proto.c line 496:#012  Condition(req->req_body_status == REQ_BODY_NONE) not true.#012version = varnish-5.2.1 revision 67e562482, vrt api = 6.1#012ident = Linux,4.15.0-10-generic,x86_64,-junix,-smalloc,-smalloc,-hcritbit,epoll#012now = 115.951166 (mono), 1519387969.491343 (real)#012Backtrace:#012  0x556fb2120826: /usr/sbin/varnishd(+0x4a826) [0x556fb2120826]#012  0x556fb216b5b0: /usr/sbin/varnishd(VAS_Fail+0x40) [0x556fb216b5b0]#012  0x556fb214a904: /usr/sbin/varnishd(+0x74904) [0x556fb214a904]#012  0x556fb213bd44: /usr/sbin/varnishd(+0x65d44) [0x556fb213bd44]#012  0x556fb213c1e0: /usr/sbin/varnishd(+0x661e0) [0x556fb213c1e0]#012  0x7f66c8ae07fc: /lib/x86_64-linux-gnu/libpthread.so.0(+0x77fc) [0x7f66c8ae07fc]#012  0x7f66c880db5f: /lib/x86_64-linux-gnu/libc.so.6(clone+0x3f) [0x7f66c880db5f]#012thread = (cache-worker)#012thr.req = 0x7f66b32e1020 {#012  vxid = 4849882, transport = H2#012  step = R_STP_LOOKUP,#012  req_body = R_BODY_WITH_LEN,#012  restarts = 0, esi_level = 0,#012  sp = 0x7f66b30b4420 {#012    fd = 320, vxid = 4849881,#012    t_open = 1519387969.434228,#012    t_idle = 1519387969.484717,#012    transport = H2 {#012      streams {#012        0x00000000 idle  #012        0x00000003 idle  #012        0x00000005 idle  #012        0x00000007 idle  #012        0x00000009 idle  #012        0x0000000b idle  #012        0x0000000d idle  #012        0x0000000f c-rem #012      }#012    }#012    client = 111.111.111.111 53766,#012    privs = 0x7f66b30b4488 {#012    },#012  },#012  ws = 0x7f66b32e1208 {#012    id = \"req\",#012    {s, f, r, e} = {0x7f66b32e3008, +1104, +57328, +57328},#012  },#012  http_conn = 0x7f66b32e1130 {#012    doclose = NULL,#012    ws = (nil) {#012    },#012    {rxbuf_b, rxbuf_e} = {(nil), (nil)},#012    {pipeline_b, pipeline_e} = {(nil), (nil)},#012    content_length = 0,#012    body_status = none,#012    first_byte_timeout = 0.000000,#012    between_bytes_timeout = 0.000000,#012  },#012  http[req] = 0x7f66b32e12a0 {#012    ws = 0x7f66b32e1208 {#012      [Already dumped, see above]#012    },#012    hdrs {#012      \"POST\",#012      \"/custom/getAllSonginfos.php\",#012      \"HTTP/2.0\",#012      \"host: webradio.ffh.de\",#012      \"scheme: https\",#012      \"user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:58.0) Gecko/20100101 Firefox/58.0\",#012      \"accept: application/json, text/javascript, */*; q=0.01\",#012      \"accept-language: de,en-US;q=0.7,en;q=0.3\",#012      \"referer: https://webradio.ffh.de/\",#012      \"x-requested-with: XMLHttpRequest\",#012      \"cookie: _ga=GA1.2.1346914010.1507966265; __gads=ID=49a7b2780b2c9b75:T=1507966266:S=ALNI_MYm49xDdLeKTZhoF8Lx-ipKRwjY2A; axd=1001412290839090112; _gid=GA1.2.1097630062.1519281984\",#012      \"content-length: 0\",#012      \"X-Forwarded-For: 62.154.206.178, 62.154.206.178\",#012      \"Accept-Encoding: gzip\",#012      \"X-UA: default\",#012    },#012  },#012  vcl = {#012    name = \"boot\",#012    busy = 203,#012    discard = 0,#012    state = auto,#012    temp = warm,#012    conf = {#012      srcname = {#012        \"/etc/varnish/default.vcl\",#012        \"Builtin\",#012        \"/etc/varnish/ffh/default.vcl\",#012        \"/etc/varnish/ffh/ffh.backends.vcl\",#012        \"/etc/varnish/ffh/ffh.recv.vcl\",#012        \"/etc/varnish/ffh/shopware.ffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/static-resource.radioteleffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/resource.radioteleffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/streams.radioteleffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/xmas.radioteleffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/livestats.radioteleffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/download.radioteleffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/static.radioteleffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/webradiostats.radioteleffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/mobile.ffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/mobile.planetradio.de/recv.vcl\",#012        \"/etc/varnish/ffh/mobile.harmonyfm.de/recv.vcl\",#012        \"/etc/varnish/ffh/mobileapps.ffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/mobileapps.planetradio.de/recv.vcl\",#012        \"/etc/varnish/ffh/mobileapps.harmonyfm.de/recv.vcl\",#012        \"/etc/varnish/ffh/newsletter.radioteleffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/webradio.ffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/webradio.planetradio.de/recv.vcl\",#012        \"/etc/varnish/ffh/webradio.harmonyfm.de/recv.vcl\",#012        \"/etc/varnish/ffh/www.ffhdigital.de/recv.vcl\",#012        \"/etc/varnish/ffh/www.radioteleffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/www.ffhacademy.de/recv.vcl\",#012        \"/etc/varnish/ffh/www.ffh.de/recv.vcl\",#012        \"/etc/varnish/ffh/www.planetradio.de/recv.vcl\",#012        \"/etc/varnish/ffh/www.harmonyfm.de/recv.vcl\",#012        \"/etc/varnish/ffh/mobileapps.wolkenkratzer-festival.de/recv.vcl\",#012        \"/etc/varnish/ffh/mobile.wolkenkratzer-festival.de/recv.vcl\",#012        \"/etc/varnish/ffh/www.wolkenkratzer-festival.de/recv.vcl\",#012        \"/etc/varnish/ffh/www.wkf.mobi/recv.vcl\",#012        \"/etc/varnish/ffh/www.nsa-hessen.de/recv.vcl\",#012        \"/etc/varnish/ffh/www.sprit-spion.de/recv.vcl\",#012        \"/etc/varnish/ffh/ffh.hit.vcl\",#012        \"/etc/varnish/ffh/ffh.miss.vcl\",#012        \"/etc/varnish/ffh/ffh.vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/shopware.ffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/static-resource.radioteleffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/resource.radioteleffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/streams.radioteleffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/xmas.radioteleffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/livestats.radioteleffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/download.radioteleffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/static.radioteleffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/webradiostats.radioteleffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/mobile.ffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/mobile.planetradio.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/mobile.harmonyfm.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/newsletter.radioteleffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/mobileapps.ffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/mobileapps.planetradio.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/mobileapps.harmonyfm.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/webradio.ffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/webradio.planetradio.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/webradio.harmonyfm.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/www.ffhdigital.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/www.radioteleffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/www.ffhacademy.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/www.ffh.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/www.planetradio.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/www.harmonyfm.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/mobile.wolkenkratzer-festival.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/mobileapps.wolkenkratzer-festival.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/www.wolkenkratzer-festival.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/www.wkf.mobi/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/www.sprit-spion.de/vcl_backend_response.vcl\",#012        \"/etc/varnish/ffh/ffh.deliver.vcl\",#012        \"/etc/varnish/ffh/resource.radioteleffh.de/deliver.vcl\",#012        \"/etc/

**Comment by gquintard** [comment]
In a more readable format:

```
Feb 23 12:07:56 cache1-vm varnishd[3140]: "Missing errorhandling code in h2_rxframe(), http2/cache_http2_proto.c line 535:
Feb 23 12:07:56 cache1-vm varnishd[3140]: Child (4547) Panic at: Fri, 23 Feb 2018 11:07:56 GMT
"Missing errorhandling code in h2_rxframe(), http2/cache_http2_proto.c line 535:
  Condition(h2->rxf_stream > h2->highest_stream) not true.thread = (cache-worker)
version = varnish-5.0.0 revision 99d036f
ident = Linux,4.13.0-36-generic,x86_64,-junix,-smalloc,-smalloc,-hcritbit,epoll
Backtrace:
  0x559691aba6df: varnishd(+0x426df) [0x559691aba6df]
  0x559691ae5091: varnishd(+0x6d091) [0x559691ae5091]
  0x559691ad4413: varnishd(+0x5c413) [0x559691ad4413]
  0x559691ad4a7b: varnishd(+0x5ca7b) [0x559691ad4a7b]
  0x7efe77ef67fc: libpthread.so.0(+0x77fc) [0x7efe77ef67fc]
  0x7efe77c23b5f: libc.so.6(clone+0x3f) [0x7efe77c23b5f]
req = 0x7efe635ff020 {
  vxid = 2228294, transport = H2
  step = 0x0,
  req_body = R_BODY_INIT,
  err_code = 1, err_reason = (null),
  restarts = 0, esi_level = 0,
  sp = 0x7efe648c3420 {
    fd = 215, vxid = 2228294,
    t_open = 1519384074.928511,
    t_idle = 1519384074.985065,
    transport = H2 {
      streams {
        0x00000000 idle  
      }
    }
client = 77.87.228.71 2467,
  },
  ws = 0x7efe635ff1f8 {
    id = \"req\",
    {s, f, r, e} = {0x7efe63600ff8, +296, (nil), +57344},
  },
  http_conn = 0x7efe635ff128 {
    fd = 215,
    doclose = NULL,
    ws = 0x7efe67154e28,
    {rxbuf_b, rxbuf_e} = {0x7efe671543f0, 0x7efe671543fd},
    {pipeline_b, pipeline_e} = {(nil), (nil)},
    content_length = 0,
    body_status = none,
    first_byte_timeout = 0.000000,
    between_bytes_timeout = 0.000000,
  },
  http[req] = 0x7efe635ff290 {
    ws[] = (nil),
    hdrs {
    },
  },
  flags = {
  },
},

"
```

And

```
Feb 23 13:12:49 cache1-vm varnishd[1094]: Child (1855) Panic at: Fri, 23 Feb 2018 12:12:49 GMT
Assert error in h2_do_req(), http2/cache_http2_proto.c line 496:
  Condition(req->req_body_status == REQ_BODY_NONE) not true.
version = varnish-5.2.1 revision 67e562482, vrt api = 6.1
ident = Linux,4.15.0-10-generic,x86_64,-junix,-smalloc,-smalloc,-hcritbit,epoll
now = 115.951166 (mono), 1519387969.491343 (real)
Backtrace:
  0x556fb2120826: /usr/sbin/varnishd(+0x4a826) [0x556fb2120826]
  0x556fb216b5b0: /usr/sbin/varnishd(VAS_Fail+0x40) [0x556fb216b5b0]
  0x556fb214a904: /usr/sbin/varnishd(+0x74904) [0x556fb214a904]
  0x556fb213bd44: /usr/sbin/varnishd(+0x65d44) [0x556fb213bd44]
  0x556fb213c1e0: /usr/sbin/varnishd(+0x661e0) [0x556fb213c1e0]
  0x7f66c8ae07fc: /lib/x86_64-linux-gnu/libpthread.so.0(+0x77fc) [0x7f66c8ae07fc]
  0x7f66c880db5f: /lib/x86_64-linux-gnu/libc.so.6(clone+0x3f) [0x7f66c880db5f]
thread = (cache-worker)
thr.req = 0x7f66b32e1020 {
  vxid = 4849882, transport = H2
  step = R_STP_LOOKUP,
  req_body = R_BODY_WITH_LEN,
  restarts = 0, esi_level = 0,
  sp = 0x7f66b30b4420 {
    fd = 320, vxid = 4849881,
    t_open = 1519387969.434228,
    t_idle = 1519387969.484717,
    transport = H2 {
      streams {
        0x00000000 idle  
        0x00000003 idle  
        0x00000005 idle  
        0x00000007 idle  
        0x00000009 idle  
        0x0000000b idle  
        0x0000000d idle  
        0x0000000f c-rem 
      }
    }
    client = 111.111.111.111 53766,
    privs = 0x7f66b30b4488 {
    },
  },
  ws = 0x7f66b32e1208 {
    id = \"req\",
    {s, f, r, e} = {0x7f66b32e3008, +1104, +57328, +57328},
  },
  http_conn = 0x7f66b32e1130 {
    doclose = NULL,
    ws = (nil) {
    },
    {rxbuf_b, rxbuf_e} = {(nil), (nil)},
    {pipeline_b, pipeline_e} = {(nil), (nil)},
    content_length = 0,
    body_status = none,
    first_byte_timeout = 0.000000,
    between_bytes_timeout = 0.000000,
  },
  http[req] = 0x7f66b32e12a0 {
    ws = 0x7f66b32e1208 {
      [Already dumped, see above]
    },
    hdrs {
      \"POST\",
      \"/custom/getAllSonginfos.php\",
      \"HTTP/2.0\",
      \"host: webradio.ffh.de\",
      \"scheme: https\",
      \"user-agent: Mozilla/5.0 (Windows NT 10.0; Win64; x64; rv:58.0) Gecko/20100101 Firefox/58.0\",
      \"accept: application/json, text/javascript, */*; q=0.01\",
      \"accept-language: de,en-US;q=0.7,en;q=0.3\",
      \"referer: https://webradio.ffh.de/\",
      \"x-requested-with: XMLHttpRequest\",
      \"cookie: _ga=GA1.2.1346914010.1507966265; __gads=ID=49a7b2780b2c9b75:T=1507966266:S=ALNI_MYm49xDdLeKTZhoF8Lx-ipKRwjY2A; axd=1001412290839090112; _gid=GA1.2.1097630062.1519281984\",
      \"content-length: 0\",
      \"X-Forwarded-For: 62.154.206.178, 62.154.206.178\",
      \"Accept-Encoding: gzip\",
      \"X-UA: default\",
    },
  },
  vcl = {
    name = \"boot\",
    busy = 203,
    discard = 0,
    state = auto,
    temp = warm,
    conf = {
      srcname = {
        \"/etc/varnish/default.vcl\",
        \"Builtin\",
        \"/etc/varnish/ffh/default.vcl\",
        \"/etc/varnish/ffh/ffh.backends.vcl\",
        \"/etc/varnish/ffh/ffh.recv.vcl\",
        \"/etc/varnish/ffh/shopware.ffh.de/recv.vcl\",
        \"/etc/varnish/ffh/static-resource.radioteleffh.de/recv.vcl\",
        \"/etc/varnish/ffh/resource.radioteleffh.de/recv.vcl\",
        \"/etc/varnish/ffh/streams.radioteleffh.de/recv.vcl\",
        \"/etc/varnish/ffh/xmas.radioteleffh.de/recv.vcl\",
        \"/etc/varnish/ffh/livestats.radioteleffh.de/recv.vcl\",
        \"/etc/varnish/ffh/download.radioteleffh.de/recv.vcl\",
        \"/etc/varnish/ffh/static.radioteleffh.de/recv.vcl\",
        \"/etc/varnish/ffh/webradiostats.radioteleffh.de/recv.vcl\",
        \"/etc/varnish/ffh/mobile.ffh.de/recv.vcl\",
        \"/etc/varnish/ffh/mobile.planetradio.de/recv.vcl\",
        \"/etc/varnish/ffh/mobile.harmonyfm.de/recv.vcl\",
        \"/etc/varnish/ffh/mobileapps.ffh.de/recv.vcl\",
        \"/etc/varnish/ffh/mobileapps.planetradio.de/recv.vcl\",
        \"/etc/varnish/ffh/mobileapps.harmonyfm.de/recv.vcl\",
        \"/etc/varnish/ffh/newsletter.radioteleffh.de/recv.vcl\",
        \"/etc/varnish/ffh/webradio.ffh.de/recv.vcl\",
        \"/etc/varnish/ffh/webradio.planetradio.de/recv.vcl\",
        \"/etc/varnish/ffh/webradio.harmonyfm.de/recv.vcl\",
        \"/etc/varnish/ffh/www.ffhdigital.de/recv.vcl\",
        \"/etc/varnish/ffh/www.radioteleffh.de/recv.vcl\",
        \"/etc/varnish/ffh/www.ffhacademy.de/recv.vcl\",
        \"/etc/varnish/ffh/www.ffh.de/recv.vcl\",
        \"/etc/varnish/ffh/www.planetradio.de/recv.vcl\",
        \"/etc/varnish/ffh/www.harmonyfm.de/recv.vcl\",
        \"/etc/varnish/ffh/mobileapps.wolkenkratzer-festival.de/recv.vcl\",
        \"/etc/varnish/ffh/mobile.wolkenkratzer-festival.de/recv.vcl\",
        \"/etc/varnish/ffh/www.wolkenkratzer-festival.de/recv.vcl\",
        \"/etc/varnish/ffh/www.wkf.mobi/recv.vcl\",
        \"/etc/varnish/ffh/www.nsa-hessen.de/recv.vcl\",
        \"/etc/varnish/ffh/www.sprit-spion.de/recv.vcl\",
        \"/etc/varnish/ffh/ffh.hit.vcl\",
        \"/etc/varnish/ffh/ffh.miss.vcl\",
        \"/etc/varnish/ffh/ffh.vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/shopware.ffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/static-resource.radioteleffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/resource.radioteleffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/streams.radioteleffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/xmas.radioteleffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/livestats.radioteleffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/download.radioteleffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/static.radioteleffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/webradiostats.radioteleffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/mobile.ffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/mobile.planetradio.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/mobile.harmonyfm.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/newsletter.radioteleffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/mobileapps.ffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/mobileapps.planetradio.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/mobileapps.harmonyfm.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/webradio.ffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/webradio.planetradio.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/webradio.harmonyfm.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/www.ffhdigital.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/www.radioteleffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/www.ffhacademy.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/www.ffh.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/www.planetradio.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/www.harmonyfm.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/mobile.wolkenkratzer-festival.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/mobileapps.wolkenkratzer-festival.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/www.wolkenkratzer-festival.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/www.wkf.mobi/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/www.sprit-spion.de/vcl_backend_response.vcl\",
        \"/etc/varnish/ffh/ffh.deliver.vcl\",
        \"/etc/varnish/ffh/resource.radioteleffh.de/deliver.vcl\",
        \"/etc/
```

**Comment by daghf** [comment]
First panic was fixed in bdf117b9176f2d65bb5d3179d298b719b310a158.

Second panic is #2305 which is fixed in 25897d362b92117888d766ec6f0b8de53e954558.

@ffh-webadmin for H/2 I would very much recommend running a varnish-weekly package (https://packagecloud.io/varnishcache/varnish-weekly). These contain both of these fixes and a number of other fixes.

Closing.
